docs/credentials.md

```diff
@@ -10,5 +10,12 @@
 1. Create a service account for your GCP project as described [here](https://cloud.google.com/iam/docs/creating-managing-service-accounts#iam-service-accounts-create-console) with the [`Compute Engine Admin role`](https://cloud.google.com/compute/docs/access/iam#compute.admin).
 2. Create a service account key for your newly service account as described [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) and export it as a JSON file.
 
+## Azure
 
+1. Create an Azure Active Directory application and it's Service Principal as described [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal). After following the previous instructions you can copy your application (client) ID and your domain, also called directory (tenant) ID.
```

Comment:
going through it: `Application (client) ID` > application (client) ID and `Directory (tenant) ID` to make it easier to pattern match with the UI

Reply:
why did we call Directory (tenant) ID the `domain`? perhaps better to call it `directoryId`? The interface also had a URI which I thought was the `domain` for a second
![Screen Shot 2021-03-02 at 8 51 59 AM](https://user-images.githubusercontent.com/2381303/109684093-c8a98200-7b34-11eb-8932-168668974e32.png)

Reply:
The nodejs client call it domain don't know why, but sure, we can call it `directoryId` or `tenantId` on the credentials file to make it easier